### PR TITLE
feat: added all links related to DSA course

### PR DESCRIPTION
**Pull request by Ciggzy1312**
Issue #53 has been fixed

Added all links related to the bootcamp :
   1. YouTube link of the Full course playlist
   2. Link to the repository for hands on practice of git
   3. Link to the repository of the whole syllabus and lectures cheatsheet

Command to check the links out : cm!dsa

### CODE REVIEW

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]
Please indent the code correctly :point_down:

**Comment by Ciggzy1312** [comment]
It is showing that my code is indented properly

**Comment by kaiwalyakoparkar** [comment]
@Ciggzy1312 I went to your repo. It still shows it's not indented. Look [here](https://github.com/Ciggzy1312/classroom-monitor-bot/blob/links/Commands/DSA.js#L19)

**Comment by Ciggzy1312** [comment]
Now check

File: Commands/DSA.js [hunk 1]
```diff
@@ -13,22 +13,22 @@ const linkEmbed = new Discord.MessageEmbed()
...
-            name: 'Complete Java + DSA course',
-            value: 'https://www.youtube.com/playlist?list=PL9gnSGHSqcnr_DxHsP7AW9ftq0AtAyYqJ',
+         	name: 'Complete Java + DSA course',
+         	value: 'https://www.youtube.com/playlist?list=PL9gnSGHSqcnr_DxHsP7AW9ftq0AtAyYqJ',
         },
-		{
-			name: 'Code sample and full syllabus for DSA lectures',
-			value: 'https://github.com/kunal-kushwaha/DSA-Bootcamp-Java',
-		},
+	{
```
Comment (kaiwalyakoparkar) [inline Commands/DSA.js:19]: See this is not yet indented :sweat_smile:

File: Commands/DSA.js [hunk 2]
```diff
@@ -13,22 +13,22 @@ const linkEmbed = new Discord.MessageEmbed()
...
+		name: 'Code sample and full syllabus for DSA lectures',
+		value: 'https://github.com/kunal-kushwaha/DSA-Bootcamp-Java',
+	},
```
Comment (kaiwalyakoparkar) [inline Commands/DSA.js:22]: This is not indented as well

File: Commands/DSA.js [hunk 3]
```diff
@@ -13,22 +13,22 @@ const linkEmbed = new Discord.MessageEmbed()
...
         {
-			name: 'Try hands on git',
-			value: 'https://github.com/kunal-kushwaha/commclassroomOP',
-		}
+		name: 'Try hands on git',
+		value: 'https://github.com/kunal-kushwaha/commclassroomOP',
+	}
```
Comment (kaiwalyakoparkar) [inline Commands/DSA.js:26]: This is not indented as well.

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

**Comment by Ciggzy1312** [comment]
image

It is showing indented in my repository :)

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket: Thanks for your collaboration and contribution

**Pull request merged**